packages/viz/src/Treemap/Treemap.tsx

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from "react";
+import ReactECharts from "echarts-for-react/lib/core";
+import { TreemapChart } from "echarts/charts";
+import { TooltipComponent } from "echarts/components";
+import * as echarts from "echarts/core";
+
+import { HvBaseChart } from "../BaseChart";
+import { HvChartTooltipClasses, useOption, useTooltip } from "../hooks";
+import { HvChartTooltip } from "../types";
+import { HvChartCommonProps } from "../types/common";
+import { HvTreemapData } from "../types/generic";
```

Comment:
This `HvTreemapData` look specific to this chart and not generic

Reply:
You're right, I'll update that.